### ISSUE TITLE
[Suggestion] GE - Set selling price with tax included

### ISSUE THREAD

**Issue by dadrek** [runelite/runelite#14461]
**Describe the solution you'd like**
It would be very beneficial to be able to automatically add the potential tax when selling an item. E.g. an extra button click/automatic feature that made it possible to specify the selling price with the taxed amount included.

*Example*
I am selling an item, and I want it to sell for 1m. Currently, if I specify this price in the G.E. I would receive 990k, as the added 1% tax results in 10k. As I want to actually receive 1m, having this feature installed, I should be able to insert the wanted amount with tax included, meaning that the selling price now is set to 1 010 101, and the received amount is 1m. See screenshots below for visual example

Basically, a feature setting the "Price per item" to `floor(price / (1 - 0.01))`

**Describe alternatives you've considered**
An alternative could be a calculator with this formula added, but I do not think that it would be that beneficial.

**Additional context**
![image](https://user-images.githubusercontent.com/95136285/145420255-423bad47-2562-43f2-8487-4900b6eac8f5.png)

![image](https://user-images.githubusercontent.com/95136285/145420409-9dc3f861-eae7-4973-adf6-7cb506a163ac.png)

![image](https://user-images.githubusercontent.com/95136285/145420199-966d29ed-de24-4849-9b0d-4d7318ea6d1a.png)
